sdk/python/generative-ai/rag/notebooks/custom_crack_and_chunk_parallel/crack_and_chunk_with_doc_intel/crack_and_chunk_parallel.py

```diff
@@ -0,0 +1,149 @@
+import os
```

Comment:
If you are using VS code to auth this script, highly recommend install Ruff extension and use its superfast formatting feature to format/beautify the python script. it also has feature to sort all your imports.